modules/mesaides/files/ma-web.conf

```diff
@@ -11,4 +11,11 @@ setuid main
 
 chdir /home/main/mes-aides-ui
 
-exec node server
+script
+    set -a
+    if [ -f /home/main/.secrets ]
+    then
+         . /home/main/.secrets
```

Comment:
Test done on `vps` instance:
I replaced `.` with `source` and restarted nginx (cf. errors) and replaced `source` with `.` (cf. UpTimeRobot requests after the errors).

```
root@vps274040:~# tail -n 20 /var/log/upstart/ma-web.log
::ffff:127.0.0.1 - - [15/Jun/2017:06:50:01 +0000] "GET / HTTP/1.1" 200 7630 "-" "curl/7.35.0"
::ffff:127.0.0.1 - - [15/Jun/2017:06:50:12 +0000] "HEAD / HTTP/1.0" 200 7630 "https://vps.mes-aides.gouv.fr" "Mozilla/5.0+(compatible; UptimeRobot/2.0; http://www.uptimerobot.com/)"
/proc/self/fd/9: 5: /proc/self/fd/9: source: not found
/proc/self/fd/9: 5: /proc/self/fd/9: source: not found
/proc/self/fd/9: 5: /proc/self/fd/9: source: not found
/proc/self/fd/9: 5: /proc/self/fd/9: source: not found
/proc/self/fd/9: 5: /proc/self/fd/9: source: not found
/proc/self/fd/9: 5: /proc/self/fd/9: source: not found
/proc/self/fd/9: 5: /proc/self/fd/9: source: not found
/proc/self/fd/9: 5: /proc/self/fd/9: source: not found
/proc/self/fd/9: 5: /proc/self/fd/9: source: not found
/proc/self/fd/9: 5: /proc/self/fd/9: source: not found
/proc/self/fd/9: 5: /proc/self/fd/9: source: not found
Mes Aides server listening on port 8000, in production mode, expecting to be deployed on https://mes-aides.gouv.fr
::ffff:127.0.0.1 - - [15/Jun/2017:06:53:14 +0000] "GET / HTTP/1.1" 200 7630 "-" "curl/7.35.0"
::ffff:127.0.0.1 - - [15/Jun/2017:06:53:23 +0000] "HEAD / HTTP/1.0" 200 7630 "https://vps.mes-aides.gouv.fr" "Mozilla/5.0+(compatible; UptimeRobot/2.0; http://www.uptimerobot.com/)"
::ffff:127.0.0.1 - - [15/Jun/2017:06:54:13 +0000] "GET / HTTP/1.1" 200 7630 "-" "curl/7.35.0"
::ffff:127.0.0.1 - - [15/Jun/2017:06:54:22 +0000] "HEAD / HTTP/1.0" 200 7630 "https://vps.mes-aides.gouv.fr" "Mozilla/5.0+(compatible; UptimeRobot/2.0; http://www.uptimerobot.com/)"
::ffff:127.0.0.1 - - [15/Jun/2017:06:55:13 +0000] "GET / HTTP/1.1" 200 7630 "-" "curl/7.35.0"
::ffff:127.0.0.1 - - [15/Jun/2017:06:55:21 +0000] "HEAD / HTTP/1.0" 200 7630 "https://vps.mes-aides.gouv.fr" "Mozilla/5.0+(compatible; UptimeRobot/2.0; http://www.uptimerobot.com/)"
```